docs/cloud-native-security/vuln-management-dashboard.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[vuln-management-dashboard]]
+// Note: This page is intentionally duplicated by docs/dashboards/vuln-management-dashboard-dash.asciidoc. When you update this page, update that page to match. And careful with the anchor links because they should not match.
+
+= Cloud Native Vulnerability Management Dashboard
+
+:frontmatter-description: The CNVM dashboard gives an overview of vulnerabilities detected in your cloud infrastructure.
+:frontmatter-tags-products: [security, cloud]
+:frontmatter-tags-content-type: [reference]
+:frontmatter-tags-user-goals: [manage]
+
+The Cloud Native Vulnerability Management (CNVM) dashboard gives you an overview of vulnerabilities detected in your cloud infrastructure.
+
+image::images/vuln-management-dashboard.png[The CNVM dashboard]
+
+.Requirements
+[sidebar]
+--
+* To collect this data, install the <<vuln-management-get-started, Cloud Native Vulnerability Management>> integration.
+* The CNVM dashboard is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
+--
+
+beta[]
+
+[[CNVM-dashboard-UI]]
+== CNVM dashboard UI
+At the top of the dashboard, five summary cards appear. They display the number of monitored cloud accounts, scanned virtual machines (VMs), and vulnerabilities (grouped by severity).
```

Comment:
Kind of a personal preference, but I avoid numbering parts of the UI — devs will add or remove things in the future, and it's easy detail to overlook. This also streamlines a little:
```suggestion
The summary cards at the top of the dashboard display the number of monitored cloud accounts, scanned virtual machines (VMs), and vulnerabilities (grouped by severity).
```